dwave_networkx/generators/chimera.py

```diff
@@ -50,13 +53,18 @@ def chimera_graph(m, n=None, t=None, create_using=None, node_list=None, edge_lis
         If provided, this graph is cleared of nodes and edges and filled
         with the new graph. Usually used to set the type of the graph.
     node_list : iterable (optional, default None)
-        Iterable of nodes in the graph. If None, calculated
-        from (m, n, t). Note that this list is used to remove nodes,
-        so any nodes specified not in ``range(m * n * 2 * t)`` are not added.
+        Iterable of nodes in the graph. If None, calculated from (``m``, ``n``, ``t``) 
+        and ``coordinates``. The nodes should be compatible with the requested 
+        coordinate system and topology bounds; by default integer-labeled
+        in :code:`range(m * n * t * 2)`. Nodes incompatible
+        with the requested topology are accepted by default.
     edge_list : iterable (optional, default None)
-        Iterable of edges in the graph. If None, edges are
-        generated as described below. The nodes in each edge must be
-        integer-labeled in ``range(m * n * t * 2)``.
+        Iterable of edges in the graph. If None, calculated
+        from the ``node_list`` as described below. 
+        Edges should be 2-tuples of nodes that match the 
+        requested coordinate system and topology bounds. Edges are accepted by 
+        default, provided component nodes are contained in the ``node_list``, 
+        otherwise they are ignored.
```

Comment:
Can we clarify here the behavior for the different types of requirements: coordinates/topology versus existing nodes? 

I seems to me that given the requirement that "provided component nodes are contained", the format is no longer relevant here; that is, I think if you say something like 

"Edges must be 2-tuples of the nodes specified in ``node_list`` or calculated from (``m``, ``n``, ``t``) and ``coordinates``; other edges are ignored unless you set ``check_edge_list`` to ``True``" 

you cover all cases, no?